migrations/Version202107270944252141_taoRevision.php

```diff
@@ -8,6 +26,7 @@
 use oat\tao\scripts\tools\accessControl\SetRolesAccess;
 use oat\tao\scripts\tools\migrations\AbstractMigration;
 
+// phpcs:disable Squiz.Classes.ValidClassName.NotCamelCaps
```

Comment:
We should run this command bellow to avoid having to add this: 

```
vendor/bin/phpcs --standard=PSR12 --extensions=php --exclude=PSR1.Classes.ClassDeclaration $dir
```

It is the same we use on the CI pipeline: https://github.com/oat-sa/tao-extension-ci-action/blob/main/action.yml#L92

Reply:
File updated, thanks :)